core/test_network/src/lib.rs

```diff
@@ -844,6 +844,7 @@ impl TestConfiguration for Configuration {
         configuration
             .load_environment()
             .expect("Failed to load configuration from environment");
+        configuration.logger.max_log_level = iroha_logger::Level::TRACE.into();
```

Comment:
BAD!

Reply:
You have the `configuration.load_environment()` use that to set the log level to `TRACE`.